K_Scaffold/Template v2/scaffold/tabs/_tabs.pug

```diff
@@ -75,7 +75,7 @@ mixin tabs({name="tabs",defaultActiveTab,persistent=true})
     //- Cleanup the class of the tab content passed through the implicit attributes
     //- and add our own internal classes
     - attributes.class = actionButtonName(replaceProblems(attributes.class || ""));
```

Comment:
Use the same ternary as above to have a space between our classes and the user-specified ones

Reply:
Newe commit to take care of this.